CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## [Unreleased]
 ### Added
 - Technical - On start, create a forestadmin-schema.json file that contains the apimap definition.
+- Technical - On production, load forestadmin-schema.json to update the apimap.
+- Technical - Allow to desactivate the automatic apimap sending on each start with the environment variable `FOREST_DISABLE_AUTO_SCHEMA_APPLY`.
```

Comment:
Change to remove we already have this feature with `FOREST_DEACTIVATE_AUTOMATIC_APIMAP` variable.